doc/en/components/charts/chart-features-navigation.md

```diff
@@ -0,0 +1,216 @@
+- `Zoom a chart to a specific level`: Use the zoom slider of the overview plus detail pane.
+- `Reset a chart to 100% zoom level`: Use the zoom reset button of the overview plus detail pane.
+- `Pan chart in all directions`: Click and drag the window rectangle inside of the preview plot area on the overview plus detail pane.
+- `Pan to a specific region of the chart`: Clicking outside of the window rectangle inside of the preview area will move the window rectangle to that area.
+- `Change mouse drag interaction in the chart`: Use the cursor button on the overview plus detail pane. This will switch between panning and drag zooming with the mouse.
+
+The following code snippet demonstrates how to enable the overview plus detail pane:
+
+```html
+// TODO
+```
+
+```tsx
+<IgrDataChart dataSource={this.data}
+    width="100%"
+    height="400px"
+    overviewPlusDetailPaneVisibility="Visible">
+</IgrDataChart>
+``` -->
+
+## Data Chart Navigation through Code
+
+The $PlatformShort$ data chart component provides several navigation properties that are updated each time a zoom or pan operation happens in the data chart. You can also set each of these properties to zoom or pan the data chart programmatically. The following is a list of these properties:
+
+- `WindowPositionHorizontal`: A numeric value describing the X portion of the content view rectangle displayed by the data chart.
+- `WindowPositionVertical`: A numeric value describing the Y portion of the content view rectangle displayed by the data chart.
+- `WindowRect`: An `IgRect` object representing a rectangle that represents the portion of the chart that is currently in view. For example, a `WindowRect` of "0, 0, 1, 1" would be the entirety of the data chart.
+- `WindowScaleHorizontal`: A numeric value describing the width portion of the content view rectangle displayed by the data chart.
+- `WindowScaleVertical`: A numeric value describing the height portion of the content view rectangle displayed by the data chart.
+
+The following code snippets show how to modify the view of the $PlatformShort$ data chart component programmatically. The following assumes you have a variable named "chart" to represent the $PlatformShort$ data chart component:
+
+```razor
+<DataChart Height="400px" Width="100%"
+    WindowScaleVertical="0.05"
+    WindowScaleHorizontal="0">
+</DataChart>                
+```
+
+```ts
+// Zoom in by a factor of 0.05
+this.chart.windowScaleVertical -= 0.05;
+this.chart.windowScaleHorizontal -= 0.05;
+// Zoom out by a factor of 0.05
+this.chart.windowScaleVertical += 0.05;
+this.chart.windowScaleHorizontal += 0.05;
+// Pan up by a factor of 0.05
+this.chart.actualWindowPositionVertical -= 0.05;
+// Pan down by a factor of 0.05
+this.chart.actualWindowPositionVertical += 0.05;
+// Pan left by a factor of 0.05
+this.chart.actualWindowPositionHorizontal -= 0.05;
+// Pan right by a factor of 0.05
+this.chart.actualWindowPositionHorizontal += 0.05;
```

Comment:
please check if "actualWindowPosition*" properties are read-only and if so please use "windowPosition*" properties

Reply:
they're not. we're good